### PR TITLE
How-Tos for multi-tier

### PR DESCRIPTION
**Pull request by jdmooney**
This PR creates How-To files for setting up a multi-tier system, with separate files for maintenance and adding a certificate.

### CODE REVIEW

**Review by sbrossie — CHANGES_REQUESTED** [review]
@jdmooney  Could you review the warnings, in particular it points to a set of sections for code blocks that seem incorrectly formatted, as per [doc](https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/) should be:


```
[source,bash]
----
```

**Comment by sbrossie** [comment]
Also, do we already have the links in place to point to these new docs, or will this come in a subsequent PR?

**Comment by jdmooney** [comment]
I think I fixed the code block links now. Still trying to get it right for Asciidoc.

As for links to these docs, I am wondering the same thing. I don't think we have a plan yet. @reshmabidikar @FlowWriter do you have any ideas?

**Comment by reshmabidikar** [comment]
> As for links to these docs, I am wondering the same thing. I don't think we have a plan yet. @reshmabidikar @FlowWriter do you have any ideas?

I believe Mary had suggested the following structure a while back:
```
Possible Reorganization:
Kill Bill on AWS
	Introduction
	Tutorials
		Single-Tier Demo
	How-To
		Setup an AWS Account
		Install Kill Bill using Cloud Formation
		Install Kill Bill using the Multi-Tier Option
		Install Kill Bill in a Serverless Container
		Add a Certificate
		Use the Simple Email Service (SES)
		Use Datadog to analyze metrics
		Use New Relic to Analyze Metrics
		Use CloudWatch to Analyze Metrics
		Use Rollbar to Track Errors
		Use Sentry to Track Errors
	Reference
		The AWS AMI
		The CloudFormation Template
		Kill Bill AWS Configuration
```

This [google doc](https://docs.google.com/document/d/1EX_Vyazg8W7mnqEuizohXOm7lPAJhNRp_9KRi2Ufh4I/edit#heading=h.5ioadz32ya8e) has the details.

File: userguide/aws/how-tos/maintain-a-multi-tier-system.adoc [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+You will need to edit this command to include:
+
+1. Your Kaui admin username and your EC2 instance ID (<ADMIN> <INSTANCE_ID>)
+2. Your database credentials (<DBUSER> <DBPASS>)
+3. The public key and secret key for your tenant (<KEY> <SECRET>)
+4. Your database URL (<DBURL>)
+
+To restrict the report to a single account, you can add the line
+
+```[source,bash]
```
[the remark this answers is not in the document]
Comment (sbrossie) [reply userguide/aws/how-tos/maintain-a-multi-tier-system.adoc]: @jdmooney Looks like there is one more to fix ^^^

**Comment by reshmabidikar** [comment]
Doesn't this PR contain the same changes as in https://github.com/killbill/killbill-docs/pull/475?

File: userguide/aws/how-tos/maintain-a-multi-tier-system.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdmooney) [reply userguide/aws/how-tos/maintain-a-multi-tier-system.adoc]: fixed

**Comment by reshmabidikar** [comment]
These changes are also included in #475. So closing this PR.

**Pull request closed without merging**